lib/service_sign_in/check-state-pension.en.yaml

```diff
@@ -20,34 +20,35 @@ choose_sign_in:
       slug: create-account
       hint_text: If you do not already have one of these accounts, we'll help you choose whether to use Government Gateway or GOV.UK Verify.
 create_new_account:
-  title: Create an account
+  title: Choose a way to prove your identity
   slug: create-account
   body: |
-    To use this service, you need to create either a Government Gateway or GOV.UK Verify account. These are used to help fight identity theft.
+    You need to confirm who you are before you can check your State Pension.
 
-    Once you have an account, you can use it to access other government services online.
+    To do this, you can create either a Government Gateway user ID or a GOV.UK Verify identity account.
 
-    ## Choose a way to prove your identity
+    ## Government Gateway
 
-    ### Government Gateway
+    You'll need:
 
-    Registering with Government Gateway usually takes about 10 minutes. It works best if you have:
+    - your National Insurance number or UK address
+    - a recent payslip or P60 or a valid UK passport
+    
+    You can use your user ID for things like your personal or business tax account, Self Assessment, Corporation Tax, PAYE for employers and VAT.
 
-    * your National Insurance number
-    * a recent payslip or P60 or a valid UK passport
+    <a role="button" class="gem-c-button govuk-button" href="https://www.tax.service.gov.uk/gg/sign-in?continue=/check-your-state-pension&accountType=individual" data-module="cross-domain-tracking" data-tracking-code="UA-43414424-1" data-tracking-name="govspeakButtonTracker" data-tracking-track-event="true">Choose Government Gateway</a>
```

Comment:
I wonder why this file uses raw HTML but the previous one uses GovSpeak? 🤔

Reply:
Yeah I thought that was strange... I've never updated these pages before and I didn't want to mess with anything I didn't need to though, so I left it alone!